plugins/scaffolder-backend/api-report.md

```diff
@@ -41,6 +41,9 @@ export type ActionContext<Input extends InputBase> = {
   input: Input;
   output(name: string, value: JsonValue): void;
   createTemporaryDirectory(): Promise<string>;
+  metadata?: {
```

Comment:
Would be nice to have this be required, if not now at least in the future. Perhaps we could add a warning if an action is being executed without metadata and say that it's deprecated? That way we can make this required in the future, which makes it a bit easier to consume

Reply:
I made it optional since I remember seeing somewhere in the docs that metadata is optional, but if that's not the way we're planning to go with the API, I can just go ahead and make it required now. Or if you'd like to go with the warning for now, where should the warning go?

Reply:
Added the warning as per our discussion.